src/App.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const todoList = [
```

Comment:
should be `items`

Reply:
Hey Dorian, thanks you for the advice. I was able to make the changes. The lesson instructed us to store it in a variable called todoList.

Reply:
I also have prettier installed, I didn't know it wasn't applying to the code. I went back to the extenttion and I ran this code in my terminal "npm install prettier -D --save-exact" is there something in the setting in which I should change

Reply:
you can do `npm install -g prettier` then `prettier --write .`